Squablr/View Controllers/SignInViewController.m

```diff
@@ -18,10 +18,6 @@ - (void)viewDidLoad {
     // Do any additional setup after loading the view.
```

Comment:
I think you should rename 'SignInViewController' to 'SignUpViewController'. 'Signing in' and 'Logging in' are synonyms which makes your current naming a little confusing since this controller signs up users (not just signs them in).